.circleci/config.yml

```diff
@@ -27,7 +27,6 @@ commands:
           command: |
             sudo apt update
             sudo apt install -y openjdk-11-jdk-headless
```

Comment:
This command should be moved to the docker image, and removed from the CircleCI workflow

Reply:
Moved it to the docker image.